src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,52 @@
+package duke;
+
+import duke.command.Command;
+
+/**
+ * Represents a Personal Assistant Chatbot that helps a person to keep track of various tasks.
+ */
+public class Duke {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
```

Comment:
You might have missed out the empty line between description and parameter section.

Reply:
Alright thanks for the reminder!